migrations/20161212123441_add-get-claim-events-by-claim-id.js

```diff
@@ -0,0 +1,46 @@
+const config = require('../config')
+
+/**
+ * Adds a table function to the IntSchema that retrieves all claim documents with claimId and grants
+ * the external web user permissions to call it.
+ *
+ */
+exports.up = function (knex, Promise) {
+  return knex.schema
+    .raw(
+      `
+        CREATE FUNCTION IntSchema.getClaimEvents(@reference varchar(7), @claimId int)
+        RETURNS TABLE
+        AS
+        RETURN
+        (
+          SELECT
+            ClaimEvent.DateAdded,
+            ClaimEvent.Caseworker,
```

Comment:
use case statement to check if this is set, do not return internal email address for caseworker